app/build.gradle

```diff
@@ -41,8 +42,5 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    testImplementation 'junit:junit:4.12'
```

Comment:
These were stubs.

I may add some happy-path tests for the CFC-specific code when I update opencv. But manual testing isn't *too* bad (since libcimbar tests the hard parts).